### DISCUSSION TITLE
Why NCP is  not on 0h 90 position (RA/Dec)?

### DISCUSSION THREAD

**Discussion by promm1983** [Stellarium/stellarium#3567] [Q&A]
Hi I'm noobie in astronomy. 
I'm just wondering why NCP in Stellarium is not on 0h 90 position and is drifting thru the time. Shouldn't it be ideally on 0h 90 degrees?

![image](https://github.com/Stellarium/stellarium/assets/21121366/ae06d6f5-5b7c-4bfd-9e24-37703038f1cc)

**Comment by gzotti** [comment]
https://en.wikipedia.org/wiki/Precession

**Comment by alex-w** [comment]
You're get the NCP from SIMBAD - https://simbad.u-strasbg.fr/simbad/sim-basic?Ident=NCP&submit=SIMBAD+search - with aberrated coordinates + Stellarium added another aberration shift.

@gzotti the second aberration is a bug and we should add antiaberration shift for objects from SIMBAD or remove aberration correction to custom objects.

**Reply by alex-w** [reply]
Done!

**Reply by gzotti** [reply]
What coordinates does SIMBAD deliver for physical objects? I had expected "Mean J2000" (catalog values without aberration). Is that in contrast coordinates including nutation, aberration (, light deflection, ...)? Is NCP or other non-physical entities handled in the same way?

**Reply by alex-w** [reply]
Good question and probably anwser in documents for SIMBAD
